annotations/openapi/oas-parameter-collectionFormat.raml

```diff
@@ -0,0 +1,10 @@
+#%RAML 1.0 AnnotationTypeDeclaration
```

Comment:
It seems that we have a hidden problem with this annotation. (In RAML we have a strict definition of how arrays of the scalars should be serialised and it seems that it depends from position where we are referencing array type) :-(